src/docs/fundamentals/ElectronicLabNotebooks.md

```diff
@@ -1,53 +1,140 @@
+| **Collaboration**     | With role management you can protect your data by limiting the access while you also are able to share the research and results to collaborate with others |
+| **Metadata**          | Document metadata in a human and machine-readable way; Read here to learn more about [Metadata][knowledge-mets]                                            |
+| **Templates**         | Templates allow you to have standardized documentation, making it easier and faster to write down measurements and results                                 |
+| **Search**            | A search function allows you to find any of your data fast and easy                                                                                        |
+| **Compliance**        | Some labs need to fulfil requirements like GLP (good laboratory practices); the ELN should therefore be compatible with the required compliance            |
+| **Evidence security** | The ELN should have an option to timestamp the individual entries, so you can trace the history of the changes made and verify the integrity of your data  |
+| **API**               | With an API, you can create interfaces for external applications like storage systems or instruments                                                       |
+| **Exit strategy**     | You should be able to export your whole ELN at any time in usable formats like PDF, zip or .eln                                                            |
+| **Reliability**       | It is preferable to use an ELN from a reliable source with long term support and development of the product                                                |
+
+For more help to find the right ELN, there is the [ELN Finder][web-elnfinder]. With it you can filter out a wide list of ELN's by different attributes.
 
 ## ARCs - Packaging ELN protocols with the data
 
 Protocols generated in ELNs integrate well with the ARC. Protocols and the metadata they contain have a dedicated place and schema within the ARC. Every study and assay (subfolder) in the ARC comes with a folder called `protocols`. This folder collects all protocols directly associated with a specific set of data files in the parallel `dataset` folder. In addition, the `isa.study.xlsx` and `isa.assay.xlsx` workbooks associated to each study or assay, help to communicate the protocol's metadata in a structured and machine-readable way.
 
 Since ARCs (are designed as FAIR digital objects, which) combine the data files with the metadata in one research data package, they facilitate to store, share and reuse the research data as a whole. <!-- rather than just the protocol // just the data without metadata  -->
 
-<!-- 
+## Example - eLabFTW <!--with LISTER-->
+
+DataPLANT is currently planning to introduce eLabFTW as an ELN solution for the community with compatibility to ISA and your ARC.
```

Comment:
I would suggest to rephrase this. 
IMO DataPLANT is not planning to introduce elabFTW. 
Since elabFTW is already used at many places, there are discussions about whether and how to extract the protocols and metadata generated there and convert it into isa.json / ARC.